Formula/deno.rb

```diff
@@ -37,6 +37,13 @@ def install
 
     # env args for building a release build with our clang, ninja and gn
     ENV["GN"] = buildpath/"gn/out/gn"
+    # Ensure build rust_v8 from the source
+    ENV["V8_FROM_SOURCE"] = "1"
+    # Chromium has decided to require the 10.15 sdk in their build config
+    ENV["FORCE_MAC_SDK_MIN"] = "10.13"
```

Comment:
Would it make sense to use `MacOS.version` here?

Reply:
Maybe, but I think it doesn't really matter here, as this only specify until which version it should complain with the 'macOS SDK not new enough'-error. It will still use the SDK provided by homebrew based on the current macOS version anyway.